### PR TITLE
My Code is About Subtraction

### PR DESCRIPTION
**Pull request by Oozegy**
This is my code about subtraction

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -1 +1,10 @@
 #It will calculate few calculations lol
+#this is from 21 life goes oin and oin and oin
+def subtraction():
+    sub1 = int(input("What is your first numberrrr"))
+    sub2 = int(input("what is your second numberrrrrrrr"))
+    if sub2 < 0:
+        sub2 = int(input("Put you second numberrr again hee hee hee haw"))
+    sub_answerr = sub1 - sub2
+    print("your answer is", sub_answerr)
+
```
Comment (IanRBr) [inline calculator.py:62]: Nice. You added the conditions as well. That's good.

**Review by IanRBr — APPROVED** [review]

File: calculator.py [hunk 2]
```diff
@@ -1 +1,10 @@
 #It will calculate few calculations lol
+#this is from 21 life goes oin and oin and oin
+def subtraction():
+    sub1 = int(input("What is your first numberrrr"))
+    sub2 = int(input("what is your second numberrrrrrrr"))
+    if sub2 < 0:
+        sub2 = int(input("Put you second numberrr again hee hee hee haw"))
+    sub_answerr = sub1 - sub2
+    print("your answer is", sub_answerr)
```
Comment (AosafRahman) [inline calculator.py]: You have to call the function

**Review by LilMarz — COMMENTED** [review]
This is good. Good job SIKE

**Review by LilMarz — APPROVED** [review]
This is good. Good job SIKE

**Review by AosafRahman — APPROVED** [review]
mike chicken butt with the mac and cheese

**Review by LilMarz — APPROVED** [review]
This is good. Good job SIKE

**Pull request merged**